### PR TITLE
WT-13634 Cut WiredTiger 11.3.0 release

### PR DESCRIPTION
**Pull request by lukech**
Changes in this PR are made to support cutting a new WiredTiger open-source release, by following the procedure documented in https://github.com/wiredtiger/wiredtiger/wiki/HOWTO%3ABuildARelease.

### CODE REVIEW

**Review by etienneptl — APPROVED** [review]

File: src/docs/upgrading.dox [hunk 1]
```diff
@@ -1,5 +1,40 @@
 /*! @page upgrading Upgrading WiredTiger applications
 
+@section version_1130 Upgrading to Version 11.3.0
+<dl>
+
+<dt>Per-file configuration "huffman_key"</dt>
```
Comment (korteland) [inline src/docs/upgrading.dox:6]: Shouldn't this be `huffman_value` since that's the thing that was changed?

File: src/docs/upgrading.dox [hunk 2]
```diff
@@ -1,5 +1,40 @@
...
+<dd>
+The checkpoint_cleanup configuration now uses a sub-configuration "method" to control
+how aggressively obsolete content is removed when creating checkpoints:
+"checkpoint_cleanup=[method=[none|reclaim_space]]"
+</dd>
+
+<dt>Connection configuration "verbose"</dt>
+<dd>
+Three eviction-related verbose config values ('evict', 'evict_stuck', and 'evictserver')
+are merged into the 'eviction' config.
```
Comment (korteland) [inline src/docs/upgrading.dox:22]: The tense here is weird, I would replace "are" with "have been".

File: src/docs/upgrading.dox [hunk 3]
```diff
@@ -1,5 +1,40 @@
...
+</dd>
+
+<dt>WT_SESSION.verify configuration "dump_app_data"</dt>
+<dd>
+The 'dump_app_data' configuration of the WT_SESSION.verify command is renamed to 'dump_all_data'.
```
Comment (korteland) [inline src/docs/upgrading.dox:27]: Same tense thing - I'd prefer "has been" over "is".

File: src/docs/upgrading.dox (hunk 1 above)
Comment (lukech) [reply src/docs/upgrading.dox]: Good catch! Addressed in [f8282ee](https://github.com/wiredtiger/wiredtiger/pull/11194/commits/f8282eeca6963f774a55f939a011acaf71258e43)

File: src/docs/upgrading.dox (hunk 2 above)
Comment (lukech) [reply src/docs/upgrading.dox]: See [f8282ee](https://github.com/wiredtiger/wiredtiger/pull/11194/commits/f8282eeca6963f774a55f939a011acaf71258e43)

File: src/docs/upgrading.dox (hunk 3 above)
Comment (lukech) [reply src/docs/upgrading.dox]: See [f8282ee](https://github.com/wiredtiger/wiredtiger/pull/11194/commits/f8282eeca6963f774a55f939a011acaf71258e43)

**Review by korteland — APPROVED** [review]
LGTM!

**Pull request merged**